examples/color/named_colors.py

```diff
@@ -1,55 +1,98 @@
 """
-========================
-Visualizing named colors
-========================
+===============
+Color reference
+===============
 
-Simple plot example with the named colors and its visual representation.
+All the named colors and what they look like.
+
+
+Matplotlib contains a number of color palettes. Here is what they look like,
+and how they are named.
 """
 from __future__ import division
 
 import matplotlib.pyplot as plt
 from matplotlib import colors as mcolors
 
 
-colors = dict(mcolors.BASE_COLORS, **mcolors.CSS4_COLORS)
+def plot_colors(colors):
+    # Sort colors by hue, saturation, value and name.
+    by_hsv = sorted(
+        (tuple(mcolors.rgb_to_hsv(mcolors.to_rgba(color)[:3])), name)
+        for name, color in colors.items())
+    sorted_names = [name for hsv, name in by_hsv]
+
+    n = len(sorted_names)
+    if n > 400:
```

Comment:
I would just stick to 4 columns everywhere, I don't think the overly wide figure at the bottom of https://2855-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/gallery/color/named_colors.html#sphx-glr-gallery-color-named-colors-py is really an improvement.

Reply:
I think I agree, it spans off my wide screen.